submissions/xT0PERx/js-dom/visuality.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Courier New", Courier, monospace;
+}
+.menu-list {
+  list-style-type: none;
+  padding: 5px 0px 5px 0px;
+  min-width: 100px;
+}
+
+.menu-list li:active {
+  background-color: rgb(223, 20, 20);
+}
+
+.menu-title {
+  padding: 20px 20px 20px 20px;
```

Comment:
`padding: 20px;` will be enought for this case).

Reply:
Fixed